mlir/test/Conversion/ArithToEmitC/arith-to-emitc.mlir

```diff
@@ -75,13 +75,18 @@ func.func @arith_integer_ops_signed_nsw(%arg0: i32, %arg1: i32) {
 // -----
```

Comment:
I don't see a test for `arith.constant : index`, could you add one please?

Reply:
There's a test for lowering of arith constants, I changed it so that it checks the type is `emitc.size_t` and doesn't check the attribute type anymore.

Reply:
Do you mean https://github.com/Xilinx/llvm-project/blob/corentin.emitc_integral_types/mlir/test/Conversion/ArithToEmitC/arith-to-emitc.mlir#L7? It doesn't seem to check for `size_t`.

Reply:
I wrote the comment before pushing the commit that did the change. Now it does :)